docs/README.md

```diff
@@ -0,0 +1,30 @@
+# Methodology Documentation
+
+This documentation leverages [Mintlify](https://mintlify.com/) to automatically generate a published version of this documetation.
+
+To read the documentation you can go to [methodology.scope3.com](https://methodology.scope3.com)
+
+### 👩‍💻 Development
+
+Install the [Mintlify CLI](https://www.npmjs.com/package/mintlify) to preview the documentation changes locally. To install, use the following command
+
+```
+npm i -g mintlify
```

Comment:
Rather than having to do this manually can you add this into the poetry config so when people do poetry install it add this for them. Keep the installation instructions all in one place

Reply:
Also can you add in the main readme line 26 can you add a link to this readme on that line. i.e. 

```
- Documentation of our calculations and assumptions in the `docs` directory. See [instructions on adding to docs](docs/README.md` 
- ```

Reply:
mintlify CLI is coming npm instead of pypi, so needs to be handled separately. I think this is ok since this is only needed if you want to preview changes and vscode has good mdx plugins for previewing mdx files.

Reply:
added note to the main README